### ISSUE TITLE
Option to echo file type in vue

### ISSUE THREAD

**Issue by ciar4n** [joomla-projects/media-manager-improvement#218]
Currently we can echo the file extension and mime however there is no server side option to echo the file type (folder/image/doc/movie etc). This would be useful in adding a type icon to the infobar and possibly each item in the list view.

Suggested types... 

- Folder
- Image
- Doc
- Video
- Audio
- Archive
- Code
- Uncategorised (default)

**Comment by laoneo** [comment]
Can this not be taken from the mime type?

**Comment by ciar4n** [comment]
I'm guessing it would be largely the same just without the subtype (**audio** instead of **audio/mpeg**)

**Comment by dneukirchen** [comment]
Its easier to extend it when this is done on server side. Its basically the same switch for client and server. 

Another option is to use naming conventions and css classes for this. Something like `.media-icon-[$EXTENSION-OR-$MIMETYPE]`

**Comment by dneukirchen** [comment]
@ciar4n what do you think about using naming conventions `.media-icon-[$EXTENSION-OR-$MIMETYPE]` ? Would that solve the problem?

**Comment by ciar4n** [comment]
Yes that could work. We can group them where needed in CSS. Extension might be best.

**Comment by dneukirchen** [comment]
Ok, thx. i will add it.
